src/core/cgroup.c

```diff
@@ -4040,11 +4040,12 @@ int unit_get_memory_current(Unit *u, uint64_t *ret) {
         return cg_get_attribute_as_uint64("memory", u->cgroup_path, r > 0 ? "memory.current" : "memory.usage_in_bytes", ret);
 }
 
-static int unit_get_memory_peak_raw(Unit *u, uint64_t *ret) {
+static int unit_get_memory_attr_raw(Unit *u, uint64_t *ret, const char* mem_attribute) {
```

Comment:
Please make `ret` the last parameter

Reply:
I am sorry, I missed that suggestion. Fixed! :)